modular_skyrat/code/modules/research/designs/autolathe_designs/autolathe_designs_sec_and_hacked.dm

```diff
@@ -10,6 +10,14 @@
 	name = "9mm Rubber Box"
 	id = "9mm_rubber"
 	build_type = AUTOLATHE
-	materials = list(/datum/material/iron = 30000)
+	materials = list(/datum/material/iron = 8000)
```

Comment:
This is a pretty massive diff

Reply:
30000 is literally 30 sheets, it's just absolutely unreasonable